Palindrome Number/main.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DefaultNamespace
+{
+    public class main
+    {
+        public bool IsPalindrome(int x)
+        {
+            if (x < 0 || (x!=0 && x % 10==0)) return false;
+            
+            int y = 0;
+            while (x > y)
+            {
+                y = y * 10 + x % 10;
+                x /= 10;
+            }
+            return (x == y || x == y/10);
```

Comment:
when will be the second condition true?

Reply:
when the count of numbers is odd, we want to compare the numbers without the middle number. 
eg. 121 => x = 1 y = 12
we want to check x == y/10,  otherwise, x==y will only return true for numbers with an even count